react/slider/src/math.ts

```diff
@@ -0,0 +1,24 @@
+export function round(value: number, precision?: number) {
+  if (!precision) {
+    return Math.round(value);
+  }
+
+  const multiplier = 10 ** (precision || 0);
+  return Math.round(value * multiplier) / multiplier;
+}
+
+export function roundIfNumber(val: string | number) {
+  if (typeof val === 'number') {
+    return round(val);
```

Comment:
this round() made me confuse with Math.round. Maybe rename it to `safeRound` or `preciseRound`?